client/src/users/Users.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import axios from "axios";
+
+import requiresAuth from "../auth/requiresAuth";
+import "../App.css";
+class Users extends React.Component {
+  state = {
+    users: []
+  };
+
+  componentDidMount() {
+    axios
+      .get("/users")
+      .then(res => {
+        this.setState({ users: res.data });
+      })
+      .catch(e => {
+        console.error(e);
+      });
```

Comment:
Since all you're doing with this `catch` is passing the parameter along to another function, you could just skip the extra step and use `console.error` as your callback:
```javascript
  .catch(console.error);
```